WORKSPACE

```diff
@@ -38,38 +38,36 @@ nodejs_register_toolchains(
     node_version = "16.9.0",
 )
 
-load("@aspect_rules_js//js:npm_import.bzl", "npm_import", "translate_package_lock")
+load("@aspect_rules_js//js:npm_import.bzl", "npm_import")
 
 # Manually import a package using explicit coordinates.
 # Just a demonstration of the syntax de-sugaring.
 npm_import(
+    name = "example_npm_deps__acorn-8.4.0",
     integrity = "sha512-ULr0LDaEqQrMFGyQ3bhJkLsbtrQ8QibAseGZeaSUiT/6zb9IvIkomWHJIvgvwad+hinRAgsI51JcWk2yvwyL+w==",
-    package = "acorn",
-    version = "8.4.0",
-    deps = [],
+    package_name = "acorn",
+    package_version = "8.4.0",
+    link_package_guard = "example",
 )
 
-# Read the package-lock.json file to automate creation of remaining npm_import rules
-translate_package_lock(
-    name = "npm_deps",
-    package_lock = "//example:package-lock.json",
+load("@aspect_rules_js//js:translate_pnpm_lock.bzl", "translate_pnpm_lock")
+
+# Read the pnpm-lock.json file to automate creation of remaining npm_import rules
+translate_pnpm_lock(
+    name = "example_npm_deps",
+    # yq -o=json -I=2 '.' pnpm-lock.yaml > pnpm-lock.json
```

Comment:
yq landed in bazel-lib so we could adopt now?

Reply:
@kormide is on it